rmgpy/chemkin.pyx

```diff
@@ -689,8 +689,8 @@ def readReactionComments(reaction, comments, read = True):
                 )
             reaction.kinetics.comment = line
 
-        elif 'ReactionLibrary:' in line or 'Seed Mechanism:' in line:
-            label = str(tokens[-1])
+        elif 'ReactionLibrary:' in line:
+            label = str(tokens[1])
```

Comment:
I think this section is for parsing RMG-Java chemkin files. You should probably leave it unchanged.